catalyst_voices/apps/voices/lib/widgets/text_field/voices_text_field.dart

```diff
@@ -162,10 +170,123 @@ class _VoicesTextFieldState extends State<VoicesTextField> {
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
     final textTheme = theme.textTheme;
+    final colors = theme.colors;
+    final colorScheme = theme.colorScheme;
 
     final labelText = widget.decoration?.labelText ?? '';
     final resizable = _isResizable;
 
+    InputDecoration buildDecoration() {
```

Comment:
Consider to make it private. Can you extract this to a separate method (avoid nesting methods)

Reply:
its nested function, accessible only inside this build

Reply:
Yep, I'm arguing that nested functions/methods are bad because you end up with more indentation than you would have if you haven't nested.